.github/workflows/ci.yml

```diff
@@ -110,10 +113,9 @@ jobs:
           mkdir build
           cd build
           cmake .. -DCMAKE_INSTALL_PREFIX=../install \
-                   -DCMAKE_C_FLAGS="-cpp --coverage -O0" \
-                   -DCMAKE_Fortran_FLAGS="-cpp --coverage -O0" \
-                   -DCMAKE_EXE_LINKER_FLAGS="--coverage" \
-                   -DCMAKE_SHARED_LINKER_FLAGS="--coverage"
+                   -DCMAKE_BUILD_TYPE=Debug \
+                   -DTESTING=ON -DJULIA_PROJECT_PATH=../libtrixi-julia \
+                   -DCOVERAGE=ON
```

Comment:
Option for turning on coverage reporting. This is nice for developers to locally get the report as well, but certainly not required for end users. I wonder if it should really be added.

The cmake module (see below) seem uncritical w.r.t. licensing.

Reply:
If it is not required for CI testing, tbh, I'd leave it out, following [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it). I don't see anyone running coverage tests locally _before_ pushing to the repo anyways. If you _do_ see a need for this, I just recommend to weigh it carefully against the potential overhead of having more code to maintain and more things that can possibly break 😅

Reply:
Sounds reasonable.

I will just try to keep it in my head.